### DISCUSSION TITLE
on reboot, it runs setup all over

### DISCUSSION THREAD

**Discussion by arslin1** [OpenVoiceOS/ovos-buildroot#183] [Q&A] [answered]
Hello-

I am new to this platform, and the documentation is still developing. I understand, and I hope this question helps others. 

On the first boot of my pi with the raspberry V0.0.8a29 (2023-07-12) image, I completed the setup. I pair the device to my account, but if I have to reboot the device, when it starts up; I have to do it all over again. 

I am likely missing something. Im sure there is a conf file that I need to modify... the documentation I have does not say anything. 

Thanks
M

**Comment by builderjer** [comment, marked as answer]
There is an issue with the `setup-skill` that causes this.  After you setup the first time, remove the `setup-skill` and it will resolve the issue.

`pip uninstall skill-ovos-setup`

or maybe

`sudo pip uninstall skill-ovos-setup`

**Comment by ChanceNCounter** [comment]
Marking the answer because we should get in that habit, and so builderjer's GitHub stats reflect his excellent customer support record =P but closing as outdated because this was fixed a while ago
